1.12.2-/common.gradle

```diff
@@ -0,0 +1,24 @@
+java {
+	sourceCompatibility = targetCompatibility = 1.8
+}
+
+repositories {
+	maven { url = "https://raw.githubusercontent.com/BleachDev/cursed-mappings/main/" }
+}
+
+base { 
+	archivesName = "${ArchiveName}-${ModVersion}"
+}
+
+dependencies {
+	modImplementation "net.fabricmc:fabric-loader:"+project.LoaderVersion
+}
```

Comment:
just a note for the future, if you're putting something into a block, say for example
```groovy
base {
  archivesName = "a"
}
```
then as long as it's just one line you can simplify it like this:
`base.archivesName = "a"`
no need for it here but I just wanted to mention it